ROADMAP.md

```diff
@@ -28,4 +8,34 @@
  
 ## Other features we are working on in the 1.7 timeframe
 
+* etcd controller to allow moving between versions
 * kops server for better team scenarios
+* support for bare-metal
+* more gossip backends
+* IAM integration
+* more cloud providers
+* promote GCE to stable
```

Comment:
Bringing rolling-update out of a alpha is a great suggestion.  WIll add.

Not sure what instance groups delete is?  We have instance-group deletion.